fcli-other/fcli-functional-test/src/ftest/groovy/com/fortify/cli/ftest/ssc/SSCPluginSpec.groovy

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ */
+package com.fortify.cli.ftest.ssc
+
+import static com.fortify.cli.ftest._common.spec.FcliSessionType.SSC
+
+import com.fortify.cli.ftest._common.Fcli
+import com.fortify.cli.ftest._common.spec.FcliBaseSpec
+import com.fortify.cli.ftest._common.spec.FcliSession
+import com.fortify.cli.ftest._common.spec.Prefix
+import com.fortify.cli.ftest.ssc._common.SSCAppVersion
+
+import spock.lang.AutoCleanup
+import spock.lang.Requires
+import spock.lang.Shared
+
+@Prefix("ssc.plugin") @FcliSession(SSC) 
+class SSCPluginSpec extends FcliBaseSpec {
+    
+    def "list"() {
+        def args = "ssc plugin list"
+        when:
+            def result = Fcli.run(args)
+        then:
+            verifyAll(result.stdout) {
+                size()>=0
+                if(size()>0) {
+                    it[0].replace(' ', '').equals("IdPluginidPlugintypePluginnamePluginversionPluginstate")
```

Comment:
Again, expect either list of plugins or 'No Data', or alternatively, make this a StepWise spec that first installs a plugin (if it's not installed already), then tests the various plugin-related commands like list, enable, disable, ..., and finally removes the plugin again (if it wasn't already installed before we started testing).

Potentially we could develop a dummy 'fcli-test' plugin, such that we can assume that this plugin will only be installed by these tests (so, usually it shouldn't exist unless a previous test run was interrupted, and it's safe to always uninstall it when we're done with the other tests)